src/materials/components/side-navigation.html

```diff
@@ -26,16 +26,28 @@ <h3 class="f-example-title">Sidebar: Single Level</h3>
 
 <section class="f-section">
 	<h3 class="f-example-title">Sidebar: Two Levels</h3>
+	<p>By default, all child menus are collapsed. To enable these by default, add the `.ds-nav-show` class. If you wish to show the class using a JavaScript interaction, add the '.ds-current' class to the child menu.</p>
 	<div class="f-example">
-		<ul class="ds-nav ds-sidebar" id="phantom-sidebar-two">
```

Comment:
@fuhton Can we keep this id for the Phantom CSS screenshot or have we updated the tests to use a different selector?